R/layers.R

```diff
@@ -91,8 +101,11 @@ emodnet_get_layers <- function(wfs = NULL, service = NULL, service_version = "2.
     # get features -------------------------------------------------------------
     out <- purrr::map2(
         .x = layers, .y = cql_filter,
-        ~ews_get_layer(.x, wfs, cql_filter = .y),
-        wfs, suppress_warnings
+        .f = function(x, y, wfs, suppress_warnings, ...){
+        ews_get_layer(x, wfs = wfs, cql_filter = y,
+                       suppress_warnings = suppress_warnings, ...)
+            },
+        wfs, suppress_warnings, ...
```

Comment:
why are the dots passed here too?

Reply:
purrr::map does funny stuff with dots so this was the only way to get it to work: https://stackoverflow.com/questions/48215325/passing-ellipsis-arguments-to-map-function-purrr-package-r

Reply:
Could you add this as code comment so we don't forget? That sounds good to know :sweat_smile: